<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCDViewerMainWindow</class>
 <widget class="QMainWindow" name="PCDViewerMainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QVTKWidget" name="qvtkWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Color_Mode">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Color_Mode"/>
    <addaction name="action_Triangulation"/>
    <addaction name="actionTour"/>
    <addaction name="actionSet_Camera"/>
    <addaction name="actionSnapshot"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Color_Mode"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="actionSnapshot"/>
   <addaction name="action_Color_Mode"/>
   <addaction name="action_Triangulation"/>
   <addaction name="actionTour"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/Open-icon.png</normaloff>:/resources/Open-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open ...</string>
   </property>
   <property name="toolTip">
    <string>Open a Point Cloud Data file (*.pcd *.ply).</string>
   </property>
   <property name="statusTip">
    <string>Click to open a Point Cloud Data file (*.pcd *.ply).</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/About-icon.png</normaloff>:/resources/About-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/Close-icon.png</normaloff>:/resources/Close-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/Save-icon.png</normaloff>:/resources/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As ...</string>
   </property>
   <property name="toolTip">
    <string>Save a copy of the data in display</string>
   </property>
   <property name="statusTip">
    <string>Click to save a copy of the data in display as a Point Cloud Data file (*.pcd *.ply).</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Color_Mode">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/Color-icon.png</normaloff>:/resources/Color-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Color Mode ...</string>
   </property>
  </action>
  <action name="action_Triangulation">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/Icon-meshes.png</normaloff>:/resources/Icon-meshes.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Meshes</string>
   </property>
   <property name="toolTip">
    <string>Add meshes to the plot.</string>
   </property>
   <property name="statusTip">
    <string>Click to configure and add.</string>
   </property>
  </action>
  <action name="actionTour">
   <property name="text">
    <string>Tour ...</string>
   </property>
  </action>
  <action name="actionSet_Camera">
   <property name="text">
    <string>Set Camera ...</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/Snapshot.png</normaloff>:/resources/Snapshot.png</iconset>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
   <property name="statusTip">
    <string>Click to take a snapshot of the plot.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
